libs/fraud/proof.go

```diff
@@ -63,7 +39,6 @@ type Proof interface {
 func OnProof(ctx context.Context, subscriber Subscriber, p ProofType, handle func(proof Proof)) {
 	subscription, err := subscriber.Subscribe(p)
 	if err != nil {
-		log.Error(err)
```

Comment:
why do you remove logs?

Reply:
Subscription does not return any other type of error besides context.Deadline etc, and having a global logger introduced just for this does not look like a idea worth playing with